### ISSUE TITLE
TAG charter needs revision for Director-free alignment

### ISSUE THREAD

**Issue by ianbjacobs** [w3c/w3process#695]
Hi all,

The TAG charter [1] includes language about the Director and will need to
be revised to align with the Director-free process.

[1] https://www.w3.org/2004/10/27-tag-charter.html

**Comment by dwsinger** [comment]
Note that there is a strong [suggestion](https://github.com/w3c/AB-memberonly/issues/49) that the TAG is defined by Process and that the charter is unmaintained and should be removed.

**Comment by fantasai** [comment]
This is an AB problem. As dsinger mentions, probably the TAG charter should be trashed in favor of the Process document.

**Comment by ianbjacobs** [comment]
Hi all,

I'm not convinced the Process Document does all the work needed for the TAG. For instance, the TAG charter discusses the patent policy obligations of the TAG (though those belong in the Patent Policy but aren't there).

Even if we argue that the Process Document takes precedence where the two are in conflict, I think there are some charter-only topics that should be revisited.

Ian

**Comment by chaals** [comment]
There was a long discussion about how the patent policy applies to the TAG, considerably after the last revision of the charter.

If there is something that should be defined for the TAG, I think it belongs in the process, not the TAG charter.

**Comment by dwsinger** [comment]
I think we decided that since only WGs can produce Recommendations, and the Patent Policy only applies to Recommendations, there is no application to the TAG.

**Comment by ianbjacobs** [comment]
The TAG has published a Recommendation:
 https://www.w3.org/TR/webarch/

**Comment by dwsinger** [comment]
> The TAG has published a Recommendation:
> https://www.w3.org/TR/webarch/

Yes, that's a known anomaly that we are quietly letting fade into history...

**Comment by ianbjacobs** [comment]
I'm hearing "Ian is to be put to pasture."

**Comment by TzviyaSiegman** [comment]
🐑

**Comment by dwsinger** [comment]
🦬
